lib/app/ui/seasons/seasons_page.dart

```diff
@@ -21,18 +19,17 @@ class _SeasonsScreenState extends State<SeasonsScreen> {
   void initState() {
     super.initState();
     WidgetsBinding.instance.addPostFrameCallback((_) {

```

Comment:
Ez picit szőrszálhasogató lehet és ízlés kérdése is, de mi egy olyan szabályt követünk, hogy a sorrend az osztályokban a következőképp alakul

- változók (először publikus, utána private) 
- override-olt függvények
- publikus függvények
- private függvények

Ennek fényében itt a helyes sorrend így néze ki: 

```
class _SeasonsScreenState extends State<SeasonsScreen> {
  late double screenWidth;
  late double screenHeight;
  
  @override
  void initState() { ... }
  
  @override
  Widget build(BuildContext context) { ... }
  
  Widget seasonCardsColumn(BuildContext context) { ... }
}
```

Reply:
Nem hinném, hogy szőrszálhasogató ez a dolog. Figyelni fogok erre!